.github/workflows/go.yaml

```diff
@@ -19,14 +19,28 @@ jobs:
         with:
           node-version: "10.x"
       - run: npm install
-  go-test:
+  lint:
```

Comment:
Since lint is only for ubuntu and not windows, did you want it in this PR or did you want to split it off into separate smaller PR?